tools/UpdateVendors/Program.cs

```diff
@@ -136,6 +144,29 @@ private static void RewriteCsFileWithStandardTransform(string filePath, string o
             Console.WriteLine($"Finished {libraryName} upgrade.");
         }
 
+        private static bool ShouldDropFile(string filePath)
+        {
+            var lowerCasePath = filePath.ToLowerInvariant();
+
+            var drops = new List<string>()
+            {
+                "StatsdClient\\DogStatsdService.cs",
+                "StatsdClient\\MetricsTimer.cs",
+                "StatsdClient\\IDogStatsD.cs",
+                "StatsdClient\\DogStatsD.cs",
+            };
+
+            foreach (var drop in drops)
+            {
+                if (lowerCasePath.Contains(drop.ToLowerInvariant()))
```

Comment:
Instead of `ToLowerInvariant()`, we should use the `String.Contains(String, StringComparison.OrdinalIgnoreCase)` overload.